consensus/src/dag/types.rs

```diff
@@ -38,6 +38,18 @@ impl TDAGMessage for CertifiedAck {
     }
 }
 
+#[derive(Clone, Serialize, Deserialize, CryptoHasher, Debug, PartialEq)]
+pub enum Extensions {
+    Empty,
```

Comment:
nit: None is probably more appropriate.

```suggestion
    None,
```